### ISSUE TITLE
suppress printing package names with no tests

### ISSUE THREAD

**Issue by renkam** [gotestyourself/gotestsum#228]
Is there a way to avoid printing packages when only skipped tests or no tests at all are in package?

**Comment by dnephin** [comment]
Currently there is no way to hide output for empty packages. Are you looking for option for the `pkgname` format , or one of the others?

**Comment by renkam** [comment]
Probably `pkgname`.
I am using `-f dots-v2` and in my project there is a lot of packages that have no tests. Printing those packages names gives me no value and pollutes the output.

**Comment by kokizzu** [comment]
I used this:
```
gotestsum -f pkgname-and-test-fails | grep -v ∅ 
# or
gotestsum -f testname | grep -v EMPTY
```
but the color is gone

**Comment by flowchartsman** [comment]
I made a PR for this, adding a `--skip-empty` option that only works if you are testing `./...`, since that's the situation where missing tests are going to show up most often. It was a bit tricky as I had to track missing stuff in two places to intercept it before it got to the formatters as well.  There might be a better method. Wasn't sure where to add tests beyond arg checking, since AFAICT all of the testing harnesses that do golden file comparison on output are geared strictly to formatters, which my solution bypasses to avoid adding skipping behavior to all formatters.
